doc/api/async_hooks.md

```diff
@@ -29,7 +29,7 @@ Following is a simple overview of the public API.
 const async_hooks = require('async_hooks');
 
 // Return the ID of the current execution context.
-const cid = async_hooks.currentId();
+const cid = async_hooks.currentAsyncId();
```

Comment:
Is `currentExecutionAsyncId()` and `currentTriggerAsyncId()` then the names? If so, may I suggest we remove the `current` part, I think it is implied from a global getter:
* `executionAsyncId()`
* `triggerAsyncId()`

> maybe async_hooks.currentContext().asyncId and async_hooks.currentContext().triggerId more make sense? But it will add a new object..

I don't think that solves anything, it just prefixes the names a bit more.